index.html

```diff
@@ -271,20 +271,20 @@ <h2 class="primary-text-about-me">About
               <div class="frame-block">    <!--Languajes-->
                 <div id="frame-skill">
                   <span class="text-skill">Languages</span>
-                  <button type="button" class="button-skill"><img src="./img/V.svg" class="V"></button>
+                  <button type="button" class="button-skill"><img src="./img/V.svg" class="V" alt=""></button>
                   <div class="separator" id="s-1"></div>
                 </div>
                 <div class="languages-list">
                   <div class="frame-list">
-                    <img class="frame-img" src="./img/Ellipse 1.svg">
+                    <img class="frame-img" src="./img/Ellipse 1.svg" alt="JavaScript skill">
                     <span class="frame-text">javaScript</span>
                   </div>
                   <div class="frame-list">
-                    <img class="frame-img" src="./img/Ellipse 1-1.svg">
+                    <img class="frame-img" src="./img/Ellipse 1-1.svg" alt="HTML skill">
                     <span class="frame-text">HTML</span>
                   </div>
```

Comment:
- [ ]  I see 😃that there is a problem with the contrast color here at the li, so kindly try to fix this contrast of all li of languages where The page has a minimum contrast ratio of at least 4.5:1 for normal-size text.😊

| The li|
| --- |
|   ![image](https://user-images.githubusercontent.com/83360636/216158002-6f8dd125-af2e-4334-9981-1ffff359c999.png) |

Reply:
> Correction

|Tags|Test contrast color|
|:---------:|:-------------------:|
|![imagen](https://user-images.githubusercontent.com/90431831/216195371-99d3e744-412f-46aa-b872-ca88a96fa415.png)|![imagen](https://user-images.githubusercontent.com/90431831/216195453-e6bb58c3-081e-464d-b1c8-de121ec89f35.png)|